script/StaticLint-juia-server.jl

```diff
@@ -0,0 +1,17 @@
+cd(@__DIR__)
+
+using Pkg;
+Pkg.activate(joinpath(pwd(),"StaticLintAtom"))
+Pkg.instantiate()
+
+# using StaticLintAtom
```

Comment:
I'm guessing this is debug/testing code? Generally commented out code shouldn't be committed.

Reply:
No, this code automatically installs the Julia dependencies